modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/api/Kobalt.kt

```diff
@@ -68,6 +68,7 @@ class Kobalt {
 
         fun addBuildFileClasspath(dep: String) {
             val dependencyManager = Kobalt.INJECTOR.getInstance(DependencyManager::class.java)
+            //FIXME buildFileClasspath collection never becomes clear! each synchronization will add the same dependencies to it
```

Comment:
@cbeust Take a look on this. Founded it while debugging... Possible solutions to use Set instead Arraylist or clear it before Build.kt processing.